src/v2/view-builder/utils/ChallengeViewUtil.js

```diff
@@ -24,7 +24,7 @@ export function doChallenge(view) {
       className: 'loopback-content',
       template: hbs`<div class="spinner"></div>`
     }));
-    view.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
+    view.doLoopback(deviceChallenge);
```

Comment:
nit: Consider adding a comment above with details on what is being passed through. This helps developers understand the `doLoopback` method interface.